plugins.txt

```diff
@@ -93,7 +93,7 @@ workflow-basic-steps:2.23
 workflow-cps:2.92
 workflow-cps-global-lib:2.19
 workflow-durable-task-step:2.39
-workflow-job:2.40
+workflow-job:2.41
```

Comment:
Same: not sure how to handle the changelog https://github.com/jenkinsci/workflow-job-plugin/releases/tag/workflow-job-2.41 let's roll